--- c-ares-1.18.1.orig/config.sub	2022-04-29 18:11:55.087420907 +0100
+++ c-ares-1.18.1/config.sub	2022-04-29 19:29:55.426670183 +0100
@@ -1364,6 +1364,7 @@
 	# Each alternative MUST end in a * to match a version number.
 	# -sysv* is not here because it comes later, after sysvr4.
 	-gnu* | -bsd* | -mach* | -minix* | -genix* | -ultrix* | -irix* \
+	      | -serenity* \
 	      | -*vms* | -sco* | -esix* | -isc* | -aix* | -cnk* | -sunos | -sunos[34]*\
 	      | -hpux* | -unos* | -osf* | -luna* | -dgux* | -auroraux* | -solaris* \
 	      | -sym* | -kopensolaris* | -plan9* \
